integreat_cms/cms/templates/pois/poi_form_sidebar/position_box.html

```diff
@@ -0,0 +1,135 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    map-pin
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Position" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div>
+        <label>{% translate "Address" %}</label>
+        <div id="nominatim-error"
+             class="bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 my-2 hidden"
+             role="alert">
+        </div>
+        <label for="{{ poi_form.address.id_for_label }}" class="secondary">{{ poi_form.address.label }}</label>
+        {% render_field poi_form.address|add_error_class:"border-red-500" %}
+        <label for="{{ poi_form.postcode.id_for_label }}" class="secondary">{{ poi_form.postcode.label }}</label>
+        {% render_field poi_form.postcode|add_error_class:"border-red-500" %}
+        <label for="{{ poi_form.city.id_for_label }}" class="secondary">{{ poi_form.city.label }}</label>
+        {% render_field poi_form.city|add_error_class:"border-red-500" %}
+        <label for="{{ poi_form.country.id_for_label }}" class="secondary">{{ poi_form.country.label }}</label>
+        {% render_field poi_form.country|add_error_class:"border-red-500" %}
+    </div>
+    <div>
+        {% render_field poi_form.location_on_map %}
+        <label for="{{ poi_form.location_on_map.id_for_label }}">{{ poi_form.location_on_map.label }}</label>
+        <div class="help-text">{{ poi_form.location_on_map.help_text }}</div>
+    </div>
+    <div>
+        <label>{% translate "Position" %}</label>
+        <div>
+            <input type="checkbox"
+                   id="auto-fill-coordinates"
+                   data-url="{% url "auto_complete_poi_address" region_slug=request.region.slug %}"
+                   checked/>
+            <label for="auto-fill-coordinates" class="secondary !inline">
+                {% translate "Derive coordinates automatically from address" %}
+            </label>
+        </div>
+        <label for="{{ poi_form.latitude.id_for_label }}" class="secondary">{{ poi_form.latitude.label }}</label>
+        {% render_field poi_form.latitude|add_error_class:"border-red-500" %}
+        <div class="help-text">{{ poi_form.latitude.help_text }}</div>
+        <label for="{{ poi_form.longitude.id_for_label }}" class="secondary">{{ poi_form.longitude.label }}</label>
+        {% render_field poi_form.longitude|add_error_class:"border-red-500" %}
+        <div class="help-text">{{ poi_form.longitude.help_text }}</div>
+    </div>
+    <div>
+        <label>{% translate "Map" %}</label>
+        <div id="map"
+             class="aspect-video"
+             data-url="{% url "get_address_from_coordinates" region_slug=request.region.slug %}"
+             data-bounding-box="{{ request.region.bounding_box.api_representation }}">
+        </div>
```

Comment:
Not sure why, but on smaller screens (where the sidebar is below the content editor), the map is flowing over:
![Screenshot 2023-01-13 at 20-58-24 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/212409302-e583ced1-c661-42e1-97ec-b2a7c20104ce.png)

Reply:
This now causes the map box to be still visible (and just blank) when collapsed:
![Screenshot 2023-01-16 at 15-22-15 Integreat Redaktionssystem](https://user-images.githubusercontent.com/16021269/212700965-6e94f61f-f82d-49d6-b40a-951f27ee56ac.png)

Reply:
Ah, I didn't think of this. I undid the change for now and will be thinking about a new solution :)

Reply:
I removed the class "flex-wrap" and this seems to solve this issue for me. I tested it for screen sizes xs to 4xl, and I think it didn't mess anything up. But still, can you please test again if this has any side effects? Thank you! :)

Reply:
Thanks, this looks very good now! :+1:
Could not find any issues on different screen resolutions.